docs/extend/legacy_plugins.md

```diff
@@ -0,0 +1,91 @@
+<!--[metadata]>
```

Comment:
Creating this content as a new file deletes its invaluable git history. I think you can use `git mv` the file to a different directory.